R/cache.R

```diff
@@ -404,7 +415,7 @@ cached_function_call <- function(fn, call, context, table, key, con, force, batc
 data_injector <- function(fcn_call, keys, cached) {
   if (length(keys) == 0) {
     return(data.frame())
-  } else if (cached) {
+  } else if (isTRUE(cached)) {
```

Comment:
Should be unnecessary since we own `data_injector` and pass a logical. But nice catch.